packages/lab/src/multiline-input/MultilineInput.tsx

```diff
@@ -0,0 +1,217 @@
+import { clsx } from "clsx";
+import {
+  ChangeEvent,
+  ComponentPropsWithoutRef,
+  FocusEvent,
+  forwardRef,
+  ReactNode,
+  Ref,
+  TextareaHTMLAttributes,
+  useState,
+} from "react";
+import { makePrefixer, StatusAdornment, useControlled } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+
+import multilineInputCss from "./MultilineInput.css";
+
+const withBaseName = makePrefixer("saltMultilineInput");
+
+export interface MultilineInputProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "defaultValue">,
+    Pick<
+      ComponentPropsWithoutRef<"input">,
+      "disabled" | "value" | "defaultValue" | "placeholder"
+    > {
+  /**
+   * The marker to use in an empty read only Input.
+   * Use `''` to disable this feature. Defaults to '—'.
+   */
+  emptyReadOnlyMarker?: string;
```

Comment:
I feel this looks a bit strange on the multiline input. Can we double check with design?

Reply:
pending @bhoppers2008

Reply:
Removed this one for now. may come back. @bhoppers2008 needs to look into it